### ISSUE TITLE
Error running example provided in README.md

### ISSUE THREAD

**Issue by StuartShanks** [NM512/dreamerv3-torch#35]
Hello, I ran the training on DMC Vision as provided in the README:
```
python3 dreamer.py --configs dmc_vision --task dmc_walker_walk --logdir ./logdir/dmc_walker_walk
```

But I got the following error:
```
Traceback (most recent call last):
  File "dreamer.py", line 386, in <module>
    main(parser.parse_args(remaining))
  File "dreamer.py", line 276, in main
    config.num_actions = acts.n if hasattr(acts, "n") else acts.shape[0]
AttributeError: 'functools.partial' object has no attribute 'shape'
```
I then lowered the number of envs from 4 (default) to 1. I could then fix the error by changing line 274 from
```
acts = train_envs[0].action_space
```
to
```
acts = train_envs[0].action_space()
```
but this resulted in other errors. I suppose the problem is elsewhere. Any help is appreciated !

**Comment by StuartShanks** [comment]
I tried changing line 274 from
```
acts = train_envs[0].action_space
```
to
```
acts = train_envs[0].action_space()
```

but this resulted in more errors. Any help appreciated !

**Comment by StuartShanks** [comment]
FIXED by downgrading gym to gym==0.19.0
